catris/views.py

```diff
@@ -513,11 +517,20 @@ def get_lines_to_render(self) -> list[bytes]:
         lines.append(b"|-------|----------|-------".ljust(80, b"-"))
 
         for hs in self._high_scores:
-            player_string = ", ".join(hs.players)
-            line_string = (
-                f"| {hs.score:<6}| {hs.get_duration_string():<9}| {player_string}"
-            )
-            line = line_string.encode("utf-8")
+            line_prefix = f"| {hs.score:<6}| {hs.get_duration_string():<9}| "
+            players = hs.players.copy()
+            while len(line_prefix + ", ".join(players)) > 80:
+                # Shorten long player names
+                new_players = []
+                maxlen = max(len(name) for name in players)
```

Comment:
Make it impossible to get stuck in infinite loop, no matter how many players there will be in the future
```suggestion
                maxlen = max(len(name) for name in players)
                assert maxlen >= 4
```